src/types/vendor/ethereumjs-wallet.d.ts

```diff
@@ -1,26 +1,15 @@
 /**
```

Comment:
Can remove?

Reply:
We don't import `ethereumjs-wallet/provider-engine`, so I removed the type.

Reply:
I meant to ask if can remove the comment.
Apologies for the ambiguity.